keyboards/hazel/bad_wings/keymaps/default/rules.mk

```diff
@@ -0,0 +1,6 @@
+MOUSEKEY_ENABLE			= yes
+EXTRAKEY_ENABLE			= yes
+CAPS_WORD_ENABLE		= yes
+TAP_DANCE_ENABLE		= yes
+TRI_LAYER_ENABLE 		= yes
+POINTING_DEVICE_ENABLE  = yes
```

Comment:
This should probably be at the kb level.

Reply:
We're talking just `POINTING_DEVICE_ENABLE`, right?  if so, that's been updated.

Reply:
I didn't mind updating the default keymap - it was mostly changes I've been wanting to make for a while.<

I've moved `TRI_LAYER_ENABLE` and `CAPS_WORD_ENABLE` to the keyboard level rules.mk. Thanks so much for your feedback!